src/redux/rocketActions.js

```diff
@@ -0,0 +1,9 @@
+export const reserveRocket = (rocketId) => ({
+  type: 'RESERVE_ROCKET',
+  payload: rocketId,
+});
+
+export const cancelReservation = (rocketId) => ({
+  type: 'CANCEL_RESERVATION',
+  payload: rocketId,
+});
```

Comment:
Following the same logic, you have to create an action creator **`fetchRockets`** in your **rocketActions.js** file to handle storing the fetched data in the Redux store. Here is an example:

```js
// Action type to store fetched rockets
export const FETCH_ROCKETS = 'FETCH_ROCKETS';

// Action creator to store fetched rockets in Redux
export const fetchRockets = (rockets) => ({
  type: FETCH_ROCKETS,
  payload: rockets,
});

// ... Rest of your code ...
```